packages.ini

```diff
@@ -1,4 +1,20 @@
 [botocore==1.25.12]
```

Comment:
These are just some test packages, right? You don't have facilities yet to update/sync this file with sentry dependency bumps (I could help with that in the near future to get some exposure to this stuff).